config/initializers/mysql.rb

```diff
@@ -4,7 +4,7 @@
   ActiveRecord::ConnectionAdapters::Mysql2Adapter::NATIVE_DATABASE_TYPES[:primary_key] = "BIGINT(20) auto_increment PRIMARY KEY"
 end
 
-if System::Database.mysql?
+unless System::Database.oracle?
```

Comment:
We are doing this for mysql and postgres now, but it is still in `config/initializers/mysql.rb` :/